### DISCUSSION TITLE
欢迎来到太微！

### DISCUSSION THREAD

**Discussion by oeyoews** [tiddly-gittly/TiddlyWiki-Chinese-Tutorial#55] [General]
:tada:

**Comment by oeyoews** [comment]
你好， 太微 :tada:

**Comment by dongrentianyu** [comment]
欢迎来到太微，你将开始一趟新的旅程，旅程很长，有些艰难，但一切都是值得的。

**Comment by fcjz** [comment]
welcome to tiddlywiki！这将是一个全新的世界。

**Comment by Zacharia2** [comment]
欢迎来到TiddlyWiki的世界！

**Comment by oeyoews** [comment]
> [太微中文官网](https://tw-cn.netlify.app/) 由于有缓存机制， 如果想要刷新到最新界面， 可以按住shift键同时点击浏览器刷新按钮

**Comment by uzvg** [comment]
awesome！
